src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/RefreshLearner/RefreshLearnerCommandHandler.cs

```diff
@@ -28,19 +27,7 @@ public async Task Handle(RefreshLearnerCommand command, CancellationToken cancel
         {
             var incentive = await _domainRepository.Find(command.ApprenticeshipIncentiveId);
 
-            var learner = await _learnerDataRepository.GetByApprenticeshipIncentiveId(incentive.Id);
-
-            if (learner == null)
-            {
-                learner = new Domain.ApprenticeshipIncentives.ValueTypes.Learner(
-                    Guid.NewGuid(), 
-                    incentive.Id, 
-                    incentive.Apprenticeship.Id, 
-                    incentive.Apprenticeship.Provider.Ukprn, 
-                    incentive.Apprenticeship.UniqueLearnerNumber, 
-                    DateTime.UtcNow
-                    );
-            }
+            var learner = await _learnerDataRepository.Get(incentive);
```

Comment:
This is out of date.  The other stories have moved on in this. This story should be branched off E-I513 or one of the other branches that hang off it.

Reply:
We are getting into a merge hell here. The branches are diverging and we need to resolve this asap. My story RI-503 is blocked until we resolve these learner conflicts